cg/meta/workflow/fastq.py

```diff
@@ -0,0 +1,275 @@
+"""
+This module handles concatenation of balsamic fastq files.
+
+Classes:
+    FastqFileNameCreator: Creates valid balsamic filenames
+    FastqHandler: Handles fastq file linking
+"""
+import datetime as dt
+import gzip
+import logging
+import os
+import re
+import shutil
+from pathlib import Path
+from typing import List, Optional
+
+LOG = logging.getLogger(__name__)
+
+DEFAULT_DATE_STR = (
+    "171015"  # Stand in value to use if sequencing date cannot be extracted from header
+)
+DEFAULT_INDEX = (
+    "XXXXXX"  # Stand in value to use if flowcell index is to be masked when renaming file
+)
+
+
+class FastqHandler:
+    """Handles fastq file linking"""
+
+    @staticmethod
+    def concatenate(files: List, concat_file: str):
+        """Concatenates a list of fastq files"""
+        LOG.info(FastqHandler.display_files(files, concat_file))
+
+        with open(concat_file, "wb") as wfd:
```

Comment:
change `wfd` to something that means something

Reply:
done